plugin.audio.intergalacticfm/README.md

```diff
@@ -0,0 +1,87 @@
+# Intergalactic FM audio plugin for Kodi
+
+![Intergalactic FM icon](resources/icon.png?raw=true)
+
+
+## Installation
+
+This add-on can be installed directly from Kodi's
+main repository and it will update itself. For more information, see
+http://kodi.wiki/view/Add-on:Intergalactic_FM
+
+Stepts to take to install from the Kodi main repository are:
+1. Go to **Add-ons** in the main menu
+2. Go to the **Add-on browser** (the box icon at the top left of the screen)
+3. Choose **Install from repository**
+4. Choose **Audio add-ons**
+5. Navigate to **Intergalactic FM** and select it
+6. Choose **Install**
+
+### Installing Kodi
+
+Instructions for installing Kodi can be found here: https://kodi.tv/download
+
+## Screenshots
+
+![Screenshot 1](screenshots/s1.png?raw=true)
```

Comment:
I think having 6 MB for 3 screenshots is a bit too much. Do they need to be 1920x1080 for a readme file?

Reply:
The screenshots are also used in the plugin info in Kodi, Kodi addon website, etc. With `optipng` we can get the total size down to 4.9 MB, is that acceptable?